lib/lib.c

```diff
@@ -87,3 +87,13 @@ bool wrmsr_safe(uint32_t msr_idx, uint64_t value) {
                  : "memory");
     return success;
 }
+
+void invlpg_safe(void *va) {
+    asm goto("1: invlpg (%[mem]); jmp %l[end];"
+             "2:" ASM_EXTABLE(1b, 2b)::[mem] "r"(va)
+             : "memory"
+             : end);
+    flush_tlb();
+end:
+    return;
+}
```

Comment:
Do we really need that? The SDM says it's supported since i486 -- hardly to find a running system equipped with an i386 or older -- less so running KTF on it ;)

Reply:
Ok, you're right. I will simplify it.